### DISCUSSION TITLE
服务器上挂载oss过一段时间出问题

### DISCUSSION THREAD

**Discussion by T-TRz879** [juicedata/juicefs#2191] [Q&A]
在服务器上挂载oss,过一段时间去看挂载点还在,但是初始化卷好像出问题了
![image](https://user-images.githubusercontent.com/50860504/172749215-69edfb4f-100c-4c2b-869f-aee8e7b13693.png)
@davies

**Comment by xiaogaozi** [comment]
你使用的元数据引擎是什么？需要检查一下元数据引擎中的数据是否还存在

**Reply by T-TRz879** [reply]
我使用的是Redis 我观察了一下Redis中,没有setting数据和文件的数据,这是怎么回事呢?
![image](https://user-images.githubusercontent.com/50860504/172751454-b197cce0-9a68-4f98-abaf-839b07d898f7.png)

**Reply by xiaogaozi** [reply]
Redis 服务有重启过吗（默认配置下数据可能没有持久化到磁盘导致重启后数据丢失）？

**Reply by T-TRz879** [reply]
我的redis是没有配置aof和rdb的,redis是放在docker中的,我去看一下日志

**Reply by T-TRz879** [reply]
> Redis 服务有重启过吗（默认配置下数据可能没有持久化到磁盘导致重启后数据丢失）？

确认Redis是没有重启过

**Reply by xiaogaozi** [reply]
你使用的 JuiceFS 的版本是？如果能稳定复现麻烦建一个 issue（需要包括复现步骤）

**Reply by T-TRz879** [reply]
我使用的是beta3,我在服务器上跑过JuiceFS挂载本地和OSS,都出现这样的问题,使用的都是docker的redis
自己的电脑也有挂载本地,使用的是普通redis,没有出现这种情况

**Reply by T-TRz879** [reply]
> 你使用的 JuiceFS 的版本是？如果能稳定复现麻烦建一个 issue（需要包括复现步骤）

我再问一个问题,就是我挂载本地的目录,redis设置了密码 format的时候出现这样的问题
![image](https://user-images.githubusercontent.com/50860504/172759484-612c21fa-fc69-4b23-b118-e0810bf1dc7e.png)

**Reply by xiaogaozi** [reply]
你需要把密码也写到 Redis 的 URL 里，详细请参考文档：https://juicefs.com/docs/zh/community/databases_for_metadata#%E5%88%9B%E5%BB%BA%E6%96%87%E4%BB%B6%E7%B3%BB%E7%BB%9F

**Reply by T-TRz879** [reply]
> 你需要把密码也写到 Redis 的 URL 里，详细请参考文档：https://juicefs.com/docs/zh/community/databases_for_metadata#%E5%88%9B%E5%BB%BA%E6%96%87%E4%BB%B6%E7%B3%BB%E7%BB%9F

谢谢:)
